keyboards/0xcb/tutelpad/readme.md

```diff
@@ -0,0 +1,15 @@
+# TutelPad macropad
+
+* Keyboard Maintainer: [ItsFiremanSam](https://github.com/ItsFiremanSam)
+* Hardware Supported: [TutelPad GitHub](https://github.com/0xCB-dev/0xCB-TutelPad)
+* Hardware Availability: [KeebSupply](https://keeb.supply/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Added an instruction for using the reset button or shorting the RST and GND pads on the controller!